### ISSUE TITLE
Some confusion about this great work.

### ISSUE THREAD

**Issue by ShihuaHuang95** [dongzhang89/FPT#35]
Many thanks for your great work. And I have read your excellent paper, but I got a few confusion after reading. 
 
1. It is kind of you to release the codes, but I could not find any trained model corresponding to the results stated in the paper. Could you please provide the corresponding trained weights?
 
2. As you have provided the detectron pre-trained models, and the [Mask RCNN with R50+FPN](https://github.com/facebookresearch/detectron2/blob/main/MODEL_ZOO.md)  in detectron2 achieved Box AP 38.6 and Mask AP 35.2, which is much higher than the baseline provided in your paper. I am wondering which baseline you have used?

**Comment by dongzhang89** [comment]
@ShihuaHuang95 Thanks your attention.
1. Thanks for your suggestions.
2. Baseline is based on BFP, not FPN.
